src/client/actions/SignUpActions.js

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import * as types from '../types/types';
+
+export const willCreateMerchant = () => {
+  return {
+    type: types.CREATE_MERCHANT
+  };
+};
+
+export const createMerchantSuccess = (merchant) => {
+  return {
+    type: types.CREATE_MERCHANT_SUCCESS,
+    merchant
+  };
+};
+
+export const createMerchantFailure = (error) => {
+  return {
+    type: types.CREATE_MERCHANT_FAILURE,
+    error
+  };
+};
+
+export const createMerchant = (merchant) => {
```

Comment:
### Nitpick
Export `createMerchant` method as default and quit exporting the rest of the actions.